src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1197,7 +1197,7 @@ private static T GetService<T>(this IServiceProvider services, T defaultService)
                     expandArguments: NoArgExpansion,
                     replaceBlankValues: ReplaceBlankWithZero,
                     checkRuntimeTypes: ExactValueTypeOrBlank<NumberValue>,
-                    checkRuntimeValues: PositiveNumericNumberChecker,
+                    checkRuntimeValues: DeferRuntimeTypeChecking,
```

Comment:
>DeferRuntimeTypeChecking [](http://example.com/codeflow?start=40&length=24)

Why? Do we want to support Sqrt(Time)? 
[you could add a comment]

Reply:
The runtime type check should take care of different values being passed to it. And we should be able to support Sqrt(Time) - after all, a Time value can be coerced to a number - so `Sqrt(Time(6,0,0))` should be `0.5` (since the numeric value of 06:00:00 is 0.25)